Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM fedora:33
```

Comment:
Can we pull from registry.fedoraproject.org instead of docker hub?

Reply:
yes we can. I can make that change